stylesheets/scss/_layout.scss

```diff
@@ -2,3 +2,45 @@
 	Layout goes in this file. Susy setup should go here too.
 	Layout classes should be prefixed `.l-`
 */
+
+// Main Configuration options for the Susy Grid
+$gutter-width: 20px;
+$column-count: 12;
+
+$susy: (
+  columns: $column-count,
+  use-custom: (rem:true),
+  global-box-sizing: border-box,
```

Comment:
Pretty sure this is used to determine how stuff is calculated.  This setting is called in `bleed` `span` and `gallery`
http://susydocs.oddbird.net/en/latest/settings/#global-box-sizing
```
~/code/susy/sass/susy/language/susy on master
$ grep -ri 'global-box-sizing' *
_bleed.scss:  @if susy-get(global-box-sizing) != content-box {
_box-sizing.scss:@mixin global-box-sizing(
_box-sizing.scss:  $box: susy-get(global-box-sizing),
_box-sizing.scss:  @include susy-inspect(global-box-sizing, $inspect);
_box-sizing.scss:  @include global-box-sizing(border-box, $inherit);
_box-sizing.scss:  @if $box != susy-get(global-box-sizing) {
_box-sizing.scss:  @include susy-set(global-box-sizing, $box);
_gallery.scss:  $global-box   : if(susy-get(global-box-sizing) == 'border-box', true, false);
_settings.scss:  global-box-sizing: content-box,
_span.scss:  $content-box: if(susy-get(global-box-sizing) != 'border-box', true, false);
```